pilot/pkg/model/validation.go

```diff
@@ -39,9 +39,9 @@ import (
 
 const (
 	dns1123LabelMaxLength int    = 63
-	dns1123LabelFmt       string = "[a-zA-Z0-9]([-a-z-A-Z0-9]*[a-zA-Z0-9])?"
```

Comment:
This is a bug

Reply:
why? caps is allowed in dns labels right?

Reply:
No, A-Z is not allowed